docs/markdown/snippets/wrap_patch.md

```diff
@@ -0,0 +1,6 @@
+## Local wrap source and patch files
+
+It is now possible to use the `patch_filename` and `source_filename` value in a
```

Comment:
Can it be also added `patch_directory` to avoid usage of archiever?

Reply:
You mean a subtree with meson.build files, or actual .patch files? For the former I did https://github.com/mesonbuild/meson/pull/6920/commits/4a12c4394e8e14ea90ac263590ab24e796f84cbd, but I agree allowing a tree would be better. For the latter I did that: https://github.com/mesonbuild/meson/pull/4570.

Reply:
I am about `patch_filename = some_patch.zip` archive content. Here is not need to use archives because single-file URL isn't used.

Reply:
This won't happen in this PR but may be implemented in a future PR. However, renaming `patch_filename` is highly unlikely, since this would be a breaking change with very little benefit.

Reply:
> However, renaming patch_filename is highly unlikely, since this would be a breaking change with very little benefit.

Why? (As I understand) `patch_filename` is introduced in this PR

Reply:
No, they already exist: https://mesonbuild.com/Wrap-dependency-system-manual.html#wrap-format